app/controllers/lookups_controller.rb

```diff
@@ -5,13 +5,18 @@ def by_base_path
     states = %w(published unpublished)
     base_paths = params.fetch(:base_paths)
 
-    base_paths_and_content_ids = Edition.with_document
-      .left_outer_joins(:unpublishing)
+    scope = Edition.left_outer_joins(:unpublishing)
+
+    # where not in (..) does not return records that where the field is null
+    scope = scope.where(unpublishings: { type: nil }).or(
+      scope.where.not(unpublishings: { type: params.fetch(:exclude_unpublishing_types, %w{vanish redirect gone}) })
+    )
+
+    scope = scope.with_document
       .where(state: states, base_path: base_paths)
-      .where("state = 'published' OR unpublishings.type = 'withdrawal'")
```

Comment:
This below line ensures that we don't get `draft` or `superseded` content in the returned results.
```
.where(state: states, base_path: base_paths)
```